### ISSUE TITLE
Debug logger

### ISSUE THREAD

**Issue by AuditeMarlow** [vswarte/gostatus#34]
When something goes horribly wrong, we just get to see an `exit 2` status code where gostatus should be. It's difficult to debug this without knowing what went wrong in the first place.

Suggesting we should add a logger that keeps what's going on during the session. Maybe timestamp it when a `panic` occurs, overwrite it when nothing goes wrong.

**Comment by vswarte** [comment]
Sure thing, let's keep a log in `/var/log`. I suggest we use [github.com/sirupsen/logrus](https://github.com/sirupsen/logrus) as it seems to be the standard.

**Comment by vswarte** [comment]
Obviously, `/var/log` requires `root` permissions. Thus we should write to the home dir. Not sure what the norm is when it comes to this.
